continuationsPlugin/src/main/scala/continuations/ContinuationsPlugin.scala

```diff
@@ -1,34 +1,48 @@
 package continuations
 
-import dotty.tools.dotc.report
-import dotty.tools.dotc.ast.Trees.*
 import dotty.tools.dotc.ast.tpd
-import dotty.tools.dotc.core.Constants.Constant
 import dotty.tools.dotc.core.Contexts.Context
-import dotty.tools.dotc.core.Decorators.*
-import dotty.tools.dotc.core.StdNames.*
-import dotty.tools.dotc.core.Symbols.*
-import dotty.tools.dotc.core.Types.{AppliedType, Type}
-import dotty.tools.dotc.plugins.{PluginPhase, StandardPlugin}
-import dotty.tools.dotc.semanticdb.TypeMessage.SealedValue.TypeRef
-import dotty.tools.dotc.transform.{PickleQuotes, Staging}
+import dotty.tools.dotc.core.Types.Type
+import dotty.tools.dotc.plugins.PluginPhase
+import dotty.tools.dotc.plugins.StandardPlugin
+import dotty.tools.dotc.report
+import dotty.tools.dotc.transform.PickleQuotes
+import dotty.tools.dotc.transform.Staging
 
 import scala.annotation.tailrec
 
 class ContinuationsPlugin extends StandardPlugin:
+
+  // test
   val name: String = "continuations"
   override val description: String = "CPS transformations"
 
   def init(options: List[String]): List[PluginPhase] =
     (new ContinuationsPhase) :: Nil
 
-class ContinuationsPhase extends PluginPhase:
+class ContinuationsPhase extends PluginPhase {
   import tpd.*
 
-  val phaseName = "continuations"
+  override def phaseName: String = "continuations"
+  override def changesBaseTypes: Boolean = true
+  override def changesMembers: Boolean = true
 
   override val runsAfter = Set(Staging.name)
   override val runsBefore = Set(PickleQuotes.name)
+  override def prepareForDefDef(tree: tpd.DefDef)(using Context): Context = summon[Context]
```

Comment:
not sure if you have seen it but there is the `dotty.tools.dotc.core.Contexts.ctx` inlined method that does the same (more or less)

Reply:
I actually don't think I need to do this here. We aren't adding anything to the context. It's vestigal from debugging. Removing.

Reply:
Resolved in https://github.com/47deg/TBD/pull/53/commits/a4ccd1d7a19ecfd29c3d2e627183f9a77957d671